script/openqa-clone-custom-git-refspec

```diff
@@ -127,7 +127,7 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
         local old_needledir
         old_needledir=$(echo "$json_data" | jq -r '.NEEDLES_DIR | select (.!=null)') || throw_json_error "$json_url" "$json_data"
         local needles_dir="${needles_dir:-"$old_needledir"}"
-        needles_dir="${needles_dir:-"$old_productdir/needles"}"
+        needles_dir="${needles_dir:-"needles"}"
```

Comment:
can we keep the script in the old form around a bit longer so that we have a chance of rolling out the os-autoinst and openQA server changes first before rolling out new scripts that only work with newer os-autoinst and server?

Reply:
I think we can keep this script in the old form around a bit longer if we keep that check
```
if (   $vars{NEEDLES_DIR}
        && !File::Spec->file_name_is_absolute($vars{NEEDLES_DIR})
        && $vars{NEEDLES_DIR} !~ /http(s)?\:\/\//)
    {
        if (my $error = _link_repo("$productdir/needles", $pooldir, $vars{NEEDLES_DIR})) { return $error }
    }
```

Reply:
good, let's keep this then and you can change the script back to the old version

Reply:
I have changed the script back to the old version. Just a comment, cloning a job (run on a remote worker) still has the problem if we keep this script as the old version.